CharlieBackend.Business/Services/MentorService.cs

```diff
@@ -220,23 +219,43 @@ public async Task<Result<MentorDto>> GetMentorByIdAsync(long mentorId)
         }
 
 
-        public async Task<Result<MentorDto>> DisableMentorAsync(long mentorId)
+        public async Task<Result<bool>> DisableMentorAsync(long mentorId)
+        {
+            var accountId = await GetAccountId(mentorId);
+
+            if (accountId == null)
+            {
+                return Result<bool>.GetError(ErrorCode.NotFound, "Unknown mentor id.");
+            }
+
+            var disable = await _accountService.DisableAccountAsync(accountId.Value);
```

Comment:
Same as here: https://github.com/ita-social-projects/WhatBackend/pull/420/files#r582779251
( `disable` -> `changedToDisabled` )

Reply:
Solved in every Service.
https://github.com/ita-social-projects/WhatBackend/pull/420/commits/bf199cebc01ab55898f5b273a77fcc44863eed4f